engine/src/main/java/org/camunda/bpm/engine/repository/ProcessDefinitionQuery.java

```diff
@@ -60,6 +62,16 @@
    */
   ProcessDefinitionQuery deploymentId(String deploymentId);
 
+  /**
+   * Only select process definitions that were deployed after the given Date (exclusive).
+   */
+  ProcessDefinitionQuery deployedAfter(Date deployedAfter);
```

Comment:
In all other APIs, `XXAfter` means that it is inclusive. This new option is exclusive. I suggest calling the method `deployedAfterExclusive` then, to make clear that this deviates from the convention.

Reply:
I found a lot of examples where `XXAfter` is used for exclusive filtering.

- ExternalTask.lockExpirationAfter
- Deployment.deploymentAfter
- Job.createdAfter
- Task.dueAfter

From a users and semantic perspective, I think this makes sense as `X` is not after `Y` if `X = Y`.